docs/source/line-sweep.rst

```diff
@@ -0,0 +1,11 @@
+Line Sweep
+=====
```

Comment:
The line with '=' (equals) should match the size of the title
Example:
"Title Length"
"========="